src/components/HelpScoutBeacon/BeaconHeadScripts.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Helmet } from 'react-helmet'
+import { GU } from '../../utils'
+
+const BEACON_EMBED =
+  '!function(e,t,n){function a(){var e=t.getElementsByTagName("script")[0],n=t.createElement("script");n.type="text/javascript",n.async=!0,n.src="https://beacon-v2.helpscout.net",e.parentNode.insertBefore(n,e)}if(e.Beacon=n=function(t,n,a){e.Beacon.readyQueue.push({method:t,options:n,data:a})},n.readyQueue=[],"complete"===t.readyState)return a();e.attachEvent?e.attachEvent("onload",a):e.addEventListener("load",a,!1)}(window,document,window.Beacon||function(){});'
+const HELPSCOUT_ID = "'163e0284-762b-4e2d-b3b3-70a73a7e6c9f'"
```

Comment:
We should probably make this an environment variable.

Reply:
It is not a secret, in the end it is part of your frontend if that is your concern.

Reply:
It’s more that it’s a dynamic part, that (IMO) belong outside of the source code. For example, we should be able to change it without releasing a new version, or we might want to let people changing it for their own (e.g. in case of a fork).

Reply:
Do we have a mechanism for something like this? Or set up manually adding the `env` var to the build process and dev?

Reply:
We do! https://github.com/aragon/aragon/blob/master/src/local-settings.js

Maybe we could also not render if the variable is not set?